sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -177,32 +186,50 @@ def get_leakage_correction(
 
         if is_new_x:
             _, acceptance_rate = utils.sample_posterior_within_prior(
-                self.neural_net, self._prior, x, num_samples=num_rejection_samples
+                self.neural_net,
+                self._prior,
+                x,
+                num_samples=num_rejection_samples,
+                show_progressbar=show_progressbar,
```

Comment:
Shouldnt we warn about high leakage here as well?

Reply:
Good point, yes!